### PR TITLE
Allow same methods

### PR DESCRIPTION
**Pull request by lzap**
Hello there,

when you have a same method in two different services, there is a conflict and OpenAPI does not validate. This fixes it.

Would you mind tagging a new release, I want to test out my RIDL if this is actually something I can start using for real :-D

### CODE REVIEW

**Comment by lzap** [comment]
Hello @VojtechVitek what is up.

**Review by VojtechVitek — APPROVED** [review]
LGTM

**Comment by VojtechVitek** [comment]
Thanks Lukas :)

File: _examples/openapi.gen.yaml [hunk 1]
```diff
@@ -1,6 +1,6 @@
-# example v0.0.1 21006c2dc2dc334a0382a645c1103910d151bb8c
+# example v0.0.1 6be99be4827f3c6b11044637ff6c61e667d58643
 # --
-# Code generated by webrpc-gen@v0.11.2 with custom generator; DO NOT EDIT
+# Code generated by webrpc-gen@v0.11.x-dev with custom generator; DO NOT EDIT
```
Comment (VojtechVitek) [inline _examples/openapi.gen.yaml]:
```suggestion
# Code generated by webrpc-gen@v0.11.2 with custom generator; DO NOT EDIT
```

**Comment by lzap** [comment]
Thanks a bunch, I am almost there for my use case. https://github.com/webrpc/gen-openapi/pull/12

**Pull request merged**